core/src/main/scala/io/finch/Trace.scala

```diff
@@ -97,12 +97,14 @@ object Trace {
    *   Trace.capture { foo(Request()).map(_ => Trace.captured) }
    * }}}
    */
+  @deprecated("Use Bootstrap.compile together with Kleisli middleware to get an access to Trace", "0.28.0")
```

Comment:
Do you think we should deprecate it? These seem to do the trick for people wanting to stick with Twitter Futures and Finagle Filters. I think it's fine if we keep them for now.

Reply:
fair enough